### DISCUSSION TITLE
How to auto shift the text accordingly when it expands

### DISCUSSION THREAD

**Discussion by starxyn** [mantinedev/mantine#6852] [Q&A]
I have a Badge placed at the most right of the Card component. However, the text inside the badge may be very long, so how do I ensure that this text will auto shift to the left without overflow?

**Comment by Kenzo-Wada** [comment]
You can use `Flex` with `justify="flex-end"`

```js
<Card>
  <Flex justify="flex-end">
    <Badge>badge</Badge>
  </Flex>
</Card>
